client/src/components/organisms/AdminRequestGroupBrowser.tsx

```diff
@@ -0,0 +1,119 @@
+import { bindActionCreators, Dispatch } from "redux"
+import { Col, Row } from "react-bootstrap";
+import { gql, useQuery } from "@apollo/client";
+import React, { FunctionComponent } from "react";
+import { connect } from "react-redux";
+import { loadRequestGroup } from '../../data/actions'
+import RequestGroup from '../../data/types/requestGroup'
+import RequestTypeDropdownList from "./RequestTypeDropdownList";
+import { RootState } from '../../data/reducers'
+import { useParams } from "react-router";
+
+import { Dropdown } from "react-bootstrap"
+
+interface StateProps {
+  requestGroup: RequestGroup
+}
+
+interface DispatchProps {
+  loadRequestGroup: typeof loadRequestGroup
+}
+
+interface ParamTypes {
+    id: string
+  }
+
+type Props = StateProps & DispatchProps;
+
+
+const AdminRequestGroupBrowser: FunctionComponent<Props> = (props: React.PropsWithChildren<Props>) => {
+    const { id } = useParams<ParamTypes>();
+    const query = gql`
+    query getRequestGroup($id: ID) {
+        requestGroup(id: $id){
```

Comment:
Make sure to only query what is needed where it is needed. This will take a VERY long time to load if we have more than a hundred requests.

If possible, split these queries up to only get what is needed in the component it is needed in

Reply:
@guy-ro  this will be changed in the future as discussed